spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-sybase.sql

```diff
@@ -27,3 +27,11 @@ CREATE TABLE INT_MESSAGE_GROUP  (
 	UPDATED_DATE DATETIME DEFAULT NULL,
 	constraint MESSAGE_GROUP_PK primary key (GROUP_KEY, REGION)
 ) LOCK DATAROWS;
+
+CREATE TABLE INT_LOCK  (
```

Comment:
I think it would be better to provide scripts for `Lock` table in the separate `lock` directory.
Not all end-user will be interested in an extra table if they use only `JdbcMessageStore`.
Plus see my comment about the DATE field, when it may be better to represent it as a `long` column (`BIGINT`, `NUMBER` etc.).
With that we don't need a separate script for specific MySQL version.

Any objections ?